openapi/fleet-manager-private.yaml

```
@@ -241,6 +241,8 @@ components:
             spec:
               type: object
               properties:
+                centralYAML:
```

Could the name be a bit more explicit that it mentions it is the CR?


```suggestion
                centralCRYAML:
```

---

done